### PR TITLE
Replace poetry dev group name

### LINKED ISSUE #29: Remove deprecated poetry dependency management method
https://python-poetry.org/docs/master/managing-dependencies/

### PR DESCRIPTION
**Pull request by jseop-lim**
## Description
* Replace deprecated poetry group name in `package.toml`.

## Ref
resolve #29

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -191,7 +191,7 @@ ipython_config.py
 #   This is especially recommended for binary packages to ensure reproducibility, and is more
 #   commonly ignored for libraries.
 #   https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control
-#poetry.lock
+poetry.lock
```
Comment (Kirade) [inline .gitignore:194]: Would you split this part into another PR?
This is not related to linked issue.

**Review by Kirade — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (jseop-lim) [reply .gitignore:194]: This is a correct suggestion! Since changing the poetry group name modifies the poetry.lock file, we should first merge the PR that adds poetry.lock to .gitignore.

**Review by Kirade — DISMISSED** [review]
